src/components/common/Button/LinkBox.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import ToastButton from '../Button/ToastButton.jsx';
+import linkIcon from '../../../images/linkBrown.svg';
+import styles from './LinkBox.module.css';
+
+export default function LinkBox() {
+  const location = useLocation();
+  const [toast, setToast] = useState(false);
+  const TOAST_TIME = 5000;
+
+  const handleCopyClipBoard = async () => {
+    await navigator.clipboard.writeText(`http://localhost:5173/${location.pathname}`);
```

Comment:
```suggestion
    await navigator.clipboard.writeText(`http://localhost:5173${location.pathname}`);
```
올려주신 영상 봤을 때 클립보드에 http://localhost:5173// 이렇게 나오더군요!
앞에 / 빼주면 올바르게 될 것 같습니다.

그리고 http://localhost:5173 과 같은 local url은 다른 곳에서도 사용할 수 있는 부분이기 때문에
변수로 빼서 다른 파일에서 관리하는 것도 좋을 것 같아요!

Reply:
제가 참고한 자료들에서 토스트의 위치 조정은 페이지 구현에서 다룬 경우가 많아 이렇게 제작했었는데, 채연님과 멘토님 이야기를 들어보니 수정해야할 것 같습니다! 로컬 URL 부분도 수정하겠습니다

Reply:
저희 해당 부분 baseUrl prop으로 바꾸는걸 깜빡했네요 ㅎㅎ